ext/microprofile/mp-rest-client/pom.xml

```diff
@@ -32,7 +32,7 @@
         <dependency>
             <groupId>org.eclipse.microprofile.rest.client</groupId>
             <artifactId>microprofile-rest-client-api</artifactId>
-            <version>1.3.3</version>
+            <version>1.4.1</version>
```

Comment:
please consider updating copyright year

Reply:
ehm... yeah, I will do that. Thank you :-)